pod_test.go

```diff
@@ -280,8 +280,8 @@ func TestPodCollector(t *testing.T) {
 				},
 			},
 			want: metadata + `
-				kube_pod_info{host_ip="1.1.1.1",namespace="ns1",pod="pod1",node="node1",pod_ip="1.2.3.4"} 1
-				kube_pod_info{host_ip="1.1.1.1",namespace="ns2",pod="pod2",node="node2",pod_ip="2.3.4.5"} 1
+				kube_pod_info{controllers="<none>",host_ip="1.1.1.1",namespace="ns1",pod="pod1",node="node1",pod_ip="1.2.3.4"} 1
```

Comment:
Agreed with `created_by`. I'm unsure but I felt like the `ownerReference` has similar if not the same information. Although by my understanding the `ownerReference` is used by the GC so if that is not the case for the `created_by` annotation then that gives the chance of writing things like the actual controller in there as in the statefulset controller or the replicaset controller created these, not a direct relation to an object in Kubernetes.